webhook/processConfig.ts

```diff
@@ -84,6 +85,19 @@ const status: Status = {
     }
   } = require(`../${PELIAS_CONFIG_FILE}`)
 
+  if (config.peliasSynonymsBase64) {
+    await updateStatus({ message: 'Importing synonyms', percentComplete: 10.0 })
+    writeFile(
+      '../pelias-config/synonyms/custom_name.txt',
+      config.peliasSynonymsBase64.replace('data:text/plain;base64,', ''),
+      { encoding: 'base64' },
+      function (err) {
+        console.log('File created')
```

Comment:
Is this console statement at the correct place?

Reply:
Could you only do `console.log` if there is an error?

Reply:
Good call! Changed